atc/db/build_test.go

```diff
@@ -2233,3 +2339,17 @@ func convertToMD5(version atc.Version) string {
 	hasher.Write([]byte(versionJSON))
 	return hex.EncodeToString(hasher.Sum(nil))
 }
+
+func getJobBuildID(col string, jobID int) int {
+	var result int
+
+	err := psql.Select(col).
+		From("jobs").
+		Where(sq.Eq{"id": jobID}).
+		RunWith(dbConn).
+		QueryRow().
+		Scan(&result)
+	Expect(err).ToNot(HaveOccurred())
+
+	return result
+}
```

Comment:
I think that this helper should not exist, for the same reason that we generally test public interfaces instead of implementation details - to help our tests remain flexible and resilient to refactors. I can imagine that the database schema that underlies these links (finished/next/transition) between builds and jobs could be refactored, say, to use a join table or a materialized view (I actually remember when the transition build idea was implemented via a materialized view). If this happens these tests will break for no good reason!

Reply:
The commit that moving from views to columns is here 2c47bef

@clarafu probably have better insights of this helper functions vs exposed interface discussion.